docs/docs/flink-configuration.md

```diff
@@ -102,30 +102,30 @@ env.getConfig()
 
 `Read option` has the highest priority, followed by `Flink configuration` and then `Table property`.
 
-| Read option                   | Flink configuration                             | Table property               | Default                          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
IDE probably did the auto reformatting. can you disable that and only change the relevant cell?

Reply:
The original table used spaces to align cell width, I followed that pattern. Added the change without reformatting